### PR TITLE
Create event page for talk at Wellcome Genome Campus

### PR DESCRIPTION
**Pull request by priyanka-surana**
- Can we add external talks about Nextflow to the events page?
- Have I entered the time correctly? It is 2pm to 3pm UK time.

### CODE REVIEW

**Comment by ewels** [comment]
@nf-core-bot fix linting

**Comment by priyanka-surana** [comment]
@nf-core-bot fix linting

File: src/content/events/2023/talk_nextflow_in_action.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
...are, nationwide, genomic diagnostic testing service, with ambitious targets of processing 300,000 samples by 2025. Currently, all clinical bioinformatics is processed using a clinical-standard certified, internally developed workflow engine (Bertha). We are migrating to a new solution (Genie) which combines Nextflow and Nextflow Tower with custom functionality, so we can focus on our core mission to enable equitably accessed, genomics medicine for all. Genie should help us support newer use cases quicker, across different infrastructures such as cloud, and uses a standard workflow definition language. We have developed an approach to migrate at speed in an agile and iterative fashion. We are mocking the services in the workflow environment in Docker containers to allow us to run continuous integration tests. We are using an automated comparison testing framework to compare the existing system with the new one to detect regressions. Later, we will iteratively refactor the workflows, breaking up the Singularity image and optimising for performance. In this talk, we will describe this migration strategy, risk management and lessons learnt while working through this large-scale effort.
```
Comment (christopher-hakkaart) [inline src/content/events/2023/talk_nextflow_in_action.md:26]:
```suggestion
## Luke Paul Buttigieg and Ricardo Humberto Ramirez Gonzalez, Genomics England

**Porting clinical genome analysis to Nextflow at Genomics England**

Genomics England provides whole genome sequencing diagnostics to the Genomic Medicine Service (U.K), a free at the point-of-care, nationwide, genomic diagnostic testing service, with ambitious targets of processing 300,000 samples by 2025. Currently, all clinical bioinformatics is processed using a clinical-standard certified, internally developed workflow engine (Bertha). We are migrating to a new solution (Genie) which combines Nextflow and Nextflow Tower with custom functionality, so we can focus on our core mission to enable equitably accessed, genomics medicine for all. Genie should help us support newer use cases quicker, across different infrastructures such as cloud, and uses a standard workflow definition language. We have developed an approach to migrate at speed in an agile and iterative fashion. We are mocking the services in the workflow environment in Docker containers to allow us to run continuous integration tests. We are using an automated comparison testing framework to compare the existing system with the new one to detect regressions. Later, we will iteratively refactor the workflows, breaking up the Singularity image and optimising for performance. In this talk, we will describe this migration strategy, risk management and lessons learnt while working through this large-scale effort.
```

File: src/content/events/2023/talk_nextflow_in_action.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+Join us for our **monthly series** of talks: **“Nextflow in Action”**.
+
+**20 minutes** for each talk + questions, we will be focussing on topics about how different organisations are using and developing with Nextflow.
+These will be recorded and made available on [YouTube](https://www.youtube.com/@workflows-community).
+It is our hope that these talks / videos will help us learn from the greater Nextflow community. If you want to present, please reach out to [Priyanka Surana](mailto:ps22@sanger.ac.uk).
+
+## Matthieu Muffato, Tree of Life, Wellcome Sanger Institute
+**Adopting Nextflow in Sanger Tree of Life**
+<br>Tree of Life is the latest scientific programme of the Wellcome Sanger Institute, aiming at producing reference genome assemblies for tens of thousands of species. We decided early on that Nextflow would be the workflow manager of choice - what an adventure it has been ! In this presentation, I will tell you about our journey, starting from training users, setting up development environments, right up to advertising our complete pipelines and reaching out to users outside of the department, with some insights into deploying and automating the pipelines.
```
Comment (christopher-hakkaart) [inline src/content/events/2023/talk_nextflow_in_action.md:22]:
```suggestion
## Matthieu Muffato, Tree of Life, Wellcome Sanger Institute

**Adopting Nextflow in Sanger Tree of Life**

Tree of Life is the latest scientific programme of the Wellcome Sanger Institute, aiming at producing reference genome assemblies for tens of thousands of species. We decided early on that Nextflow would be the workflow manager of choice - what an adventure it has been ! In this presentation, I will tell you about our journey, starting from training users, setting up development environments, right up to advertising our complete pipelines and reaching out to users outside of the department, with some insights into deploying and automating the pipelines.
```

**Review by christopher-hakkaart — CHANGES_REQUESTED** [review]

**Comment by priyanka-surana** [comment]
Standard prettier does not work for me on this file. 
```
$ prettier -w src/content/events/2023/talk_nextflow_in_action.md
[error] Invalid configuration for file "/lustre/scratch123/tol/teams/tolit/users/ps22/pipelines/website/src/content/events/2023/talk_nextflow_in_action.md":
[error] Cannot find module 'prettier-plugin-astro'
[error] Require stack:
[error] - /lustre/scratch123/tol/teams/tolit/users/ps22/pipelines/website/.prettierrc.cjs
[error] - /software/treeoflife/miniconda3/envs/nf-core_2.10/lib/node_modules/prettier/internal/internal.mjs
```
@christopher-hakkaart Let me know what needs doing to get this file merged. Thank you.

**Comment by mashehu** [comment]
@nf-core-bot fix linting

**Pull request merged**